Bottom: 3b21fbd54d4b3f98b9248eb2f0247a00d61a0176
Top:    6d568218b73580a6ddd5883d6c6f52a5279ecc90
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-09-05 15:09:28 +0800

Refresh of vhost-prefetch-vring

---

diff --git a/samples/bpf/xdp1_kern.c b/samples/bpf/xdp1_kern.c
index a4b89d3..2197421 100644
--- a/samples/bpf/xdp1_kern.c
+++ b/samples/bpf/xdp1_kern.c
@@ -51,8 +51,6 @@ int xdp_prog1(struct xdp_md *ctx)
 	u64 nh_off;
 	u32 ipproto;
 
-	return rc;
-
 	nh_off = sizeof(*eth);
 	if (data + nh_off > data_end)
 		return rc;
